jobs/atc/spec

```diff
@@ -288,6 +288,12 @@ properties:
       for and release old caches of resource versions.
     default: 30s
 
+  container_placement_strategy:
+    description: |
+      Method by which a worker is selected during container placement.
+
+      Options are "volume-locality" and "random".
```

Comment:
Could you add `default: volume-locality`?

Reply:
Done